diff --git a/DEMO/CHASE.CPP b/DEMO/CHASE.CPP
index 34a346a..cfb6e83 100644
--- a/DEMO/CHASE.CPP
+++ b/DEMO/CHASE.CPP
@@ -424,7 +424,9 @@ static void Reflected_Transform(Scene *Sc)
 	for (O = Sc->OmniHead; O; O = O->Next)
 	{
 		Vtx = &O->V;
-		Vector_Sub(&O->IPos, &View->ISource, &V);
+		Vector ReflectedPos = O->IPos;
+		ReflectedPos.y = -ReflectedPos.y;
+		Vector_Sub(&ReflectedPos, &View->ISource, &V);
 		MatrixXVector(View->Mat, &V, &Vtx->TPos);
 		if (Vtx->TPos.z > Sc->NZP&&Vtx->TPos.z < Sc->FZP)
 		{
@@ -436,7 +438,7 @@ static void Reflected_Transform(Scene *Sc)
 			//dz *=-16384;
 			//dz +=0x7FFFFFFF;
 			//RoundToInt(&O->Face.SortZ.DW,dz);
-			// *Ins++ = &O->F; // no insertion : flares are not reflected for now
+			*Ins++ = &O->F; // no insertion : flares are not reflected for now
 
 			O->F.FlareSize = O->ISize;
 #ifdef FRONT_TO_BACK_SORTING
diff --git a/DEMO/CITY.CPP b/DEMO/CITY.CPP
index c9253bb..ee58963 100644
--- a/DEMO/CITY.CPP
+++ b/DEMO/CITY.CPP
@@ -215,7 +215,7 @@ static void Reflective_Surface_Setup()
 
 			// Extract coverage (range) over XZ plane and over texture
 			// (notice: this reads texture coords. from vertices, not faces).
-/*			float mx, Mx, mz, Mz, y;
+			float mx, Mx, mz, Mz, y;
 			float mu, Mu, mv, Mv;
 			dword i, j;
 			mx = Mx = T->Verts[0].Pos.x;
@@ -273,8 +273,8 @@ static void Reflective_Surface_Setup()
 					Vtx->Pos.x = x;
 					Vtx->Pos.y = y;
 					Vtx->Pos.z = z;
-					Vtx->U = u;
-					Vtx->V = v;
+					Vtx->U = u / 2.5;
+					Vtx->V = v / 2.5;
 					Vtx++;
 				}
 			}
@@ -308,7 +308,7 @@ static void Reflective_Surface_Setup()
 					F->V3 = F->C->V;
 					F++;
 				}
-			}*/
+			}
 
 //			break;
 		} else {
@@ -726,7 +726,9 @@ OUT:FEnd=T->Faces+T->FIndex;
 	for(O=Sc->OmniHead;O;O=O->Next)
 	{
 		Vtx=&O->V;
-		Vector_Sub(&O->IPos,&View->ISource,&V);
+		Vector ReflectedPos = O->IPos;
+		ReflectedPos.y = -ReflectedPos.y;
+		Vector_Sub(&ReflectedPos,&View->ISource,&V);
 		MatrixXVector(View->Mat,&V,&Vtx->TPos);
 		if (Vtx->TPos.z>Sc->NZP&&Vtx->TPos.z<Sc->FZP)
 		{
@@ -734,11 +736,13 @@ OUT:FEnd=T->Faces+T->FIndex;
 			Vtx->PX=CntrEX+Vtx->TPos.x*PX*Vtx->RZ;
 			Vtx->PY=CntrEY-Vtx->TPos.y*PY*Vtx->RZ;
 			// Insert to List
+			//dz = Vtx->TPos.z;
+			//dz *=-16384;
+			//dz +=0x7FFFFFFF;
+			//RoundToInt((long *)&O->F.SortZ.DW,dz);
 			dz = Vtx->TPos.z;
-			dz *=-16384;
-			dz +=0x7FFFFFFF;
-			RoundToInt((long *)&O->F.SortZ.DW,dz);
 			*Ins++ = &O->F; // no insertion : flares are not reflected for now
+			O->F.SortZ.F = 2 * fzp - dz;
 		}
 	}
 	COmnies = (Ins-FList)-CPolys;
@@ -973,12 +977,12 @@ void Reflective_AnimateTexture()
 	Face *F, *FE;
 	for(F = T->Faces, FE = F + T->FIndex; F<FE; F++)
 	{
-		F->U1 += dTime * 0.0007;
-		F->V1 += dTime * 0.0003;
-		F->U2 += dTime * 0.0007;
-		F->V2 += dTime * 0.0003;
-		F->U3 += dTime * 0.0007;
-		F->V3 += dTime * 0.0003;
+		F->U1 += dTime * 0.0007 * 2;
+		F->V1 += dTime * 0.0003 * 2;
+		F->U2 += dTime * 0.0007 * 2;
+		F->V2 += dTime * 0.0003 * 2;
+		F->U3 += dTime * 0.0007 * 2;
+		F->V3 += dTime * 0.0003 * 2;
 	}
 
 }
diff --git a/DEMO/DEMO.vcxproj b/DEMO/DEMO.vcxproj
index 2f76892..7a631e8 100644
--- a/DEMO/DEMO.vcxproj
+++ b/DEMO/DEMO.vcxproj
@@ -23,7 +23,7 @@
     <SccProjectName />
     <SccLocalPath />
     <ProjectGuid>{B1AD0193-422D-9C62-DC27-DA50B9122A44}</ProjectGuid>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.10586.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
@@ -119,7 +119,7 @@
       <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
       <BufferSecurityCheck>false</BufferSecurityCheck>
       <FunctionLevelLinking>true</FunctionLevelLinking>
-      <EnableEnhancedInstructionSet>AdvancedVectorExtensions</EnableEnhancedInstructionSet>
+      <EnableEnhancedInstructionSet>AdvancedVectorExtensions2</EnableEnhancedInstructionSet>
       <PrecompiledHeader>
       </PrecompiledHeader>
       <PrecompiledHeaderOutputFile>.\Release/DEMO.pch</PrecompiledHeaderOutputFile>
@@ -137,7 +137,6 @@
       <OmitFramePointers>false</OmitFramePointers>
       <EnableParallelCodeGeneration>true</EnableParallelCodeGeneration>
       <FloatingPointExceptions>false</FloatingPointExceptions>
-      <WholeProgramOptimization>false</WholeProgramOptimization>
     </ClCompile>
     <Link>
       <AdditionalOptions>/MACHINE:I386 %(AdditionalOptions)</AdditionalOptions>
@@ -240,7 +239,7 @@
       <DebugInformationFormat>EditAndContinue</DebugInformationFormat>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
       <FloatingPointModel>Strict</FloatingPointModel>
-      <FloatingPointExceptions>true</FloatingPointExceptions>
+      <FloatingPointExceptions>false</FloatingPointExceptions>
       <ExceptionHandling>Async</ExceptionHandling>
       <AdditionalOptions>%(AdditionalOptions)</AdditionalOptions>
       <OmitFramePointers>false</OmitFramePointers>
@@ -381,4 +380,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
diff --git a/DEMO/Glat.cpp b/DEMO/Glat.cpp
index e79e60f..22ee139 100644
--- a/DEMO/Glat.cpp
+++ b/DEMO/Glat.cpp
@@ -557,25 +557,25 @@ void Run_Glato(void)
 		if (Code)
 		{
 //			Grid_Texture_Mapper_XXX(Code_GP,CodeImage,(DWord *)Page2);
-			//GridRendererT(Code_GP,CodeImage,(DWord *)Page2, XRes, YRes);
-			Grid_Texture_Mapper_T(Code_GP, CodeImage, (DWord *)Page2);
+			GridRendererT(Code_GP,CodeImage,(DWord *)Page2, XRes, YRes);
+			//Grid_Texture_Mapper_T(Code_GP, CodeImage, (DWord *)Page2);
 			Modulate(&Surf1,&Surf2,0xa0a0a0,0xa0a0a0);
-			Modulate(&Surf2,VSurface,0xa0a0a0, 0xa0a0a0);
+			Modulate(&Surf2,VSurface,0xa0a0a0, 0xb0b0b0);
 		}
 		if (Gfx)
 		{
 //			Grid_Texture_Mapper_XXX(Gfx_GP,GfxImage,(DWord *)Page3);
-			Grid_Texture_Mapper_T(Gfx_GP,GfxImage,(DWord *)Page3);
+			GridRendererT(Gfx_GP,GfxImage,(DWord *)Page3, XRes, YRes);
 			Modulate(&Surf1,&Surf3,0xa0a0a0, 0xd0d0d0);
-			Modulate(&Surf3,VSurface,0xa0a0a0, 0xa0a0a0);
+			Modulate(&Surf3,VSurface,0xa0a0a0, 0xb0b0b0);
  //			Modulate(&Surf2,&Surf3,0xa0a0a0,0xa0a0a0);
 		}
 		if (Sfx)
 		{
 //			Grid_Texture_Mapper_XXX(Sfx_GP,SfxImage,(DWord *)Page4);
-			Grid_Texture_Mapper_T(Sfx_GP,SfxImage,(DWord *)Page4);
+			GridRendererT(Sfx_GP,SfxImage,(DWord *)Page4, XRes, YRes);
 			Modulate(&Surf1,&Surf4,0xa0a0a0,0xa0a0a0);
-			Modulate(&Surf4,VSurface,0xa0a0a0, 0xa0a0a0);
+			Modulate(&Surf4,VSurface,0xa0a0a0, 0xb0b0b0);
 		}
 //		memcpy(VPage, Page1, PageSize);
 		if (Timer>3200)
diff --git a/FDS/Base/FDS_VARS.H b/FDS/Base/FDS_VARS.H
index 6ddb40d..5ea74a0 100644
--- a/FDS/Base/FDS_VARS.H
+++ b/FDS/Base/FDS_VARS.H
@@ -700,47 +700,46 @@ modify [eax ecx edx];
 template <typename MATRIX, typename VECTOR>
 inline void MatrixXVector(MATRIX M, VECTOR *U, VECTOR *V)
 {
-
 	V->x = M[0][0] * U->x + M[0][1] * U->y + M[0][2] * U->z;
 	V->y = M[1][0] * U->x + M[1][1] * U->y + M[1][2] * U->z;
 	V->z = M[2][0] * U->x + M[2][1] * U->y + M[2][2] * U->z;
-
-	//__asm {
-	//	mov	EDX, [M]
-	//		mov ESI, [U]
-	//		mov EDI, [V]
-	//		FLD   DWORD PTR [ESI]
-	//		FMUL  DWORD PTR [EDX]
-	//		FLD   DWORD PTR [ESI]
-	//		FMUL  DWORD PTR [EDX+12]
-	//		FLD   DWORD PTR [ESI]
-	//		FMUL  DWORD PTR [EDX+24]
-	//		FLD   DWORD PTR [ESI+4]
-	//		FMUL  DWORD PTR [EDX+4]
-	//		FLD   DWORD PTR [ESI+4]
-	//		FMUL  DWORD PTR [EDX+16]
-	//		FLD   DWORD PTR [ESI+4]
-	//		FMUL  DWORD PTR [EDX+28]
-	//		FXCH  ST(2)
-	//		FADDP ST(5),ST(0)
-	//		FADDP ST(3),ST(0)
-	//		FADDP ST(1),ST(0)
-	//		FLD   DWORD PTR [ESI+8]
-	//		FMUL  DWORD PTR [EDX+8]
-	//		FLD   DWORD PTR [ESI+8]
-	//		FMUL  DWORD PTR [EDX+20]
-	//		FLD   DWORD PTR [ESI+8]
-	//		FMUL  DWORD PTR [EDX+32]
-	//		FXCH  ST(2)
-	//		FADDP ST(5),ST(0)
-	//		FADDP ST(3),ST(0)
-	//		FADDP ST(1),ST(0)
-	//		FXCH  ST(1)
-	//		FXCH  ST(2)
-	//		FSTP  DWORD PTR [EDI]
-	//		FSTP  DWORD PTR [EDI+8]
-	//		FSTP  DWORD PTR [EDI+4]
-	//}
+/*
+	__asm {
+		mov	EDX, [M]
+			mov ESI, [U]
+			mov EDI, [V]
+			FLD   DWORD PTR [ESI]
+			FMUL  DWORD PTR [EDX]
+			FLD   DWORD PTR [ESI]
+			FMUL  DWORD PTR [EDX+12]
+			FLD   DWORD PTR [ESI]
+			FMUL  DWORD PTR [EDX+24]
+			FLD   DWORD PTR [ESI+4]
+			FMUL  DWORD PTR [EDX+4]
+			FLD   DWORD PTR [ESI+4]
+			FMUL  DWORD PTR [EDX+16]
+			FLD   DWORD PTR [ESI+4]
+			FMUL  DWORD PTR [EDX+28]
+			FXCH  ST(2)
+			FADDP ST(5),ST(0)
+			FADDP ST(3),ST(0)
+			FADDP ST(1),ST(0)
+			FLD   DWORD PTR [ESI+8]
+			FMUL  DWORD PTR [EDX+8]
+			FLD   DWORD PTR [ESI+8]
+			FMUL  DWORD PTR [EDX+20]
+			FLD   DWORD PTR [ESI+8]
+			FMUL  DWORD PTR [EDX+32]
+			FXCH  ST(2)
+			FADDP ST(5),ST(0)
+			FADDP ST(3),ST(0)
+			FADDP ST(1),ST(0)
+			FXCH  ST(1)
+			FXCH  ST(2)
+			FSTP  DWORD PTR [EDI]
+			FSTP  DWORD PTR [EDI+8]
+			FSTP  DWORD PTR [EDI+4]
+	}*/
 }
 
 inline void Vector_Add(Vector *U,Vector *V,Vector *W)
diff --git a/FDS/FDS.vcxproj b/FDS/FDS.vcxproj
index 3d245a3..3432552 100644
--- a/FDS/FDS.vcxproj
+++ b/FDS/FDS.vcxproj
@@ -22,7 +22,7 @@
     <SccProjectName />
     <SccLocalPath />
     <ProjectGuid>{D1ED1229-DB09-58AB-3236-DA76C00C4763}</ProjectGuid>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.10586.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
@@ -116,7 +116,7 @@
       <ObjectFileName>.\Release/</ObjectFileName>
       <ProgramDataBaseFileName>.\Release/</ProgramDataBaseFileName>
       <SuppressStartupBanner>true</SuppressStartupBanner>
-      <EnableEnhancedInstructionSet>AdvancedVectorExtensions</EnableEnhancedInstructionSet>
+      <EnableEnhancedInstructionSet>AdvancedVectorExtensions2</EnableEnhancedInstructionSet>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
       <WarningLevel>Level3</WarningLevel>
       <AdditionalOptions>/J</AdditionalOptions>
@@ -195,7 +195,7 @@
       <DebugInformationFormat>EditAndContinue</DebugInformationFormat>
       <MultiProcessorCompilation>true</MultiProcessorCompilation>
       <FloatingPointModel>Strict</FloatingPointModel>
-      <FloatingPointExceptions>true</FloatingPointExceptions>
+      <FloatingPointExceptions>false</FloatingPointExceptions>
       <ExceptionHandling>Async</ExceptionHandling>
       <AdditionalOptions>%(AdditionalOptions)</AdditionalOptions>
       <OmitFramePointers>false</OmitFramePointers>
@@ -730,4 +730,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
diff --git a/FDS/RENDER/RENDER.CPP b/FDS/RENDER/RENDER.CPP
index a4ea644..07203d0 100644
--- a/FDS/RENDER/RENDER.CPP
+++ b/FDS/RENDER/RENDER.CPP
@@ -180,11 +180,15 @@ void Animate_Objects(Scene *Sc)
 	// lalala, HARARCHIA , Ver 3, it now rulati
 	for (Obj=Sc->ObjectHead;Obj;Obj=Obj->Next)
 	{
-		if (Obj->Pivot.is_zero()) {
-			U.x = U.y = U.z = 0.0f;
-		} else {
-			MatrixXVector(*Obj->Rot, &Obj->Pivot, &U);
+		if (Obj->Type == Obj_Omni) {
+			Matrix_Identity(*Obj->Rot);
 		}
+		//if (Obj->Pivot.is_zero()) {
+		MatrixXVector(*Obj->Rot, &Obj->Pivot, &U);
+//			U.x = U.y = U.z = 0.0f;
+		//} else {
+			//MatrixXVector(*Obj->Rot, &Obj->Pivot, &U);
+		//}
 		//printf("%s U: \n", Obj->Name);
 		//U.print();
 		//printf("\n");
diff --git a/IJLD/IJLD.vcxproj b/IJLD/IJLD.vcxproj
index 3791bde..815012f 100644
--- a/IJLD/IJLD.vcxproj
+++ b/IJLD/IJLD.vcxproj
@@ -22,7 +22,7 @@
     <SccProjectName />
     <SccLocalPath />
     <ProjectGuid>{8B62AF28-C242-9B32-27DF-492A187C8D42}</ProjectGuid>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.10586.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
@@ -228,4 +228,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
diff --git a/Runtime/City.stat b/Runtime/City.stat
deleted file mode 100644
index 7a45e9b..0000000
--- a/Runtime/City.stat
+++ /dev/null
@@ -1,402 +0,0 @@
-Width Stats:
-	4.800000	0
-	14.400001	5017
-	24.000000	1335
-	33.600002	0
-	43.200001	0
-	52.800003	0
-	62.400002	0
-	72.000000	0
-	81.600006	0
-	91.200005	0
-	100.800003	0
-	110.400002	0
-	120.000000	0
-	129.600006	0
-	139.200012	0
-	148.800003	0
-	158.400009	0
-	168.000000	0
-	177.600006	0
-	187.200012	0
-	196.800003	0
-	206.400009	0
-	216.000000	0
-	225.600006	0
-	235.200012	0
-	244.800003	0
-	254.400009	0
-	264.000000	0
-	273.600006	0
-	283.200012	0
-	292.800018	0
-	302.400024	0
-	312.000000	0
-	321.600006	0
-	331.200012	0
-	340.800018	0
-	350.400024	0
-	360.000000	0
-	369.600006	0
-	379.200012	0
-	388.800018	0
-	398.400024	0
-	408.000000	0
-	417.600006	0
-	427.200012	0
-	436.800018	0
-	446.400024	0
-	456.000000	0
-	465.600006	0
-	475.200012	0
-	484.800018	0
-	494.400024	0
-	504.000000	0
-	513.600037	0
-	523.200012	0
-	532.800049	0
-	542.400024	0
-	552.000000	0
-	561.600037	0
-	571.200012	0
-	580.800049	0
-	590.400024	0
-	600.000000	0
-	609.600037	0
-	619.200012	0
-	628.800049	0
-	638.400024	0
-	648.000000	0
-	657.600037	0
-	667.200012	0
-	676.800049	0
-	686.400024	0
-	696.000000	0
-	705.600037	0
-	715.200012	0
-	724.800049	0
-	734.400024	0
-	744.000000	0
-	753.600037	0
-	763.200012	0
-	772.800049	0
-	782.400024	0
-	792.000000	0
-	801.600037	0
-	811.200012	0
-	820.800049	0
-	830.400024	0
-	840.000000	0
-	849.600037	0
-	859.200012	0
-	868.800049	0
-	878.400024	0
-	888.000000	0
-	897.600037	0
-	907.200012	0
-	916.800049	0
-	926.400024	0
-	936.000000	0
-	945.600037	0
-	955.200012	0
-	964.800049	0
-	974.400024	0
-	984.000000	0
-	993.600037	0
-	1003.200012	0
-	1012.800049	0
-	1022.400024	0
-	1032.000000	0
-	1041.600098	0
-	1051.200073	0
-	1060.800049	0
-	1070.400024	0
-	1080.000000	0
-	1089.600098	0
-	1099.200073	0
-	1108.800049	0
-	1118.400024	0
-	1128.000000	0
-	1137.600098	0
-	1147.200073	0
-	1156.800049	0
-	1166.400024	0
-	1176.000000	0
-	1185.600098	0
-	1195.200073	0
-	1204.800049	0
-	1214.400024	0
-	1224.000000	0
-	1233.600098	0
-	1243.200073	0
-	1252.800049	0
-	1262.400024	0
-	1272.000000	0
-	1281.600098	0
-	1291.200073	0
-	1300.800049	0
-	1310.400024	0
-	1320.000000	0
-	1329.600098	0
-	1339.200073	0
-	1348.800049	0
-	1358.400024	0
-	1368.000000	0
-	1377.600098	0
-	1387.200073	0
-	1396.800049	0
-	1406.400024	0
-	1416.000000	0
-	1425.600098	0
-	1435.200073	0
-	1444.800049	0
-	1454.400024	0
-	1464.000000	0
-	1473.600098	0
-	1483.200073	0
-	1492.800049	0
-	1502.400024	0
-	1512.000000	0
-	1521.600098	0
-	1531.200073	0
-	1540.800049	0
-	1550.400024	0
-	1560.000000	0
-	1569.600098	0
-	1579.200073	0
-	1588.800049	0
-	1598.400024	0
-	1608.000000	0
-	1617.600098	0
-	1627.200073	0
-	1636.800049	0
-	1646.400024	0
-	1656.000000	0
-	1665.600098	0
-	1675.200073	0
-	1684.800049	0
-	1694.400024	0
-	1704.000000	0
-	1713.600098	0
-	1723.200073	0
-	1732.800049	0
-	1742.400024	0
-	1752.000000	0
-	1761.600098	0
-	1771.200073	0
-	1780.800049	0
-	1790.400024	0
-	1800.000000	0
-	1809.600098	0
-	1819.200073	0
-	1828.800049	0
-	1838.400024	0
-	1848.000000	0
-	1857.600098	0
-	1867.200073	0
-	1876.800049	0
-	1886.400024	0
-	1896.000000	0
-	1905.600098	0
-	1915.200073	0
-Height Stats:
-	2.700000	0
-	8.100000	1151
-	13.500000	433
-	18.900002	68
-	24.300001	0
-	29.700001	0
-	35.100002	0
-	40.500000	0
-	45.900002	0
-	51.300003	0
-	56.700001	0
-	62.100002	0
-	67.500000	0
-	72.900002	0
-	78.300003	0
-	83.700005	0
-	89.100006	0
-	94.500000	0
-	99.900002	0
-	105.300003	0
-	110.700005	0
-	116.100006	0
-	121.500000	0
-	126.900002	0
-	132.300003	0
-	137.700012	0
-	143.100006	0
-	148.500000	0
-	153.900009	0
-	159.300003	0
-	164.700012	0
-	170.100006	0
-	175.500000	0
-	180.900009	0
-	186.300003	0
-	191.700012	0
-	197.100006	0
-	202.500000	0
-	207.900009	0
-	213.300003	0
-	218.700012	0
-	224.100006	0
-	229.500000	0
-	234.900009	0
-	240.300003	0
-	245.700012	0
-	251.100006	0
-	256.500000	0
-	261.900024	0
-	267.300018	0
-	272.700012	0
-	278.100006	0
-	283.500000	0
-	288.900024	0
-	294.300018	0
-	299.700012	0
-	305.100006	0
-	310.500000	0
-	315.900024	0
-	321.300018	0
-	326.700012	0
-	332.100006	0
-	337.500000	0
-	342.900024	0
-	348.300018	0
-	353.700012	0
-	359.100006	0
-	364.500000	0
-	369.900024	0
-	375.300018	0
-	380.700012	0
-	386.100006	0
-	391.500000	0
-	396.900024	0
-	402.300018	0
-	407.700012	0
-	413.100006	0
-	418.500000	0
-	423.900024	0
-	429.300018	0
-	434.700012	0
-	440.100006	0
-	445.500000	0
-	450.900024	0
-	456.300018	0
-	461.700012	0
-	467.100006	0
-	472.500000	0
-	477.900024	0
-	483.300018	0
-	488.700012	0
-	494.100006	0
-	499.500000	0
-	504.900024	0
-	510.300018	0
-	515.700012	0
-	521.100037	0
-	526.500000	0
-	531.900024	0
-	537.300049	0
-	542.700012	0
-	548.100037	0
-	553.500000	0
-	558.900024	0
-	564.300049	0
-	569.700012	0
-	575.100037	0
-	580.500000	0
-	585.900024	0
-	591.300049	0
-	596.700012	0
-	602.100037	0
-	607.500000	0
-	612.900024	0
-	618.300049	0
-	623.700012	0
-	629.100037	0
-	634.500000	0
-	639.900024	0
-	645.300049	0
-	650.700012	0
-	656.100037	0
-	661.500000	0
-	666.900024	0
-	672.300049	0
-	677.700012	0
-	683.100037	0
-	688.500000	0
-	693.900024	0
-	699.300049	0
-	704.700012	0
-	710.100037	0
-	715.500000	0
-	720.900024	0
-	726.300049	0
-	731.700012	0
-	737.100037	0
-	742.500000	0
-	747.900024	0
-	753.300049	0
-	758.700012	0
-	764.100037	0
-	769.500000	0
-	774.900024	0
-	780.300049	0
-	785.700012	0
-	791.100037	0
-	796.500000	0
-	801.900024	0
-	807.300049	0
-	812.700012	0
-	818.100037	0
-	823.500000	0
-	828.900024	0
-	834.300049	0
-	839.700012	0
-	845.100037	0
-	850.500000	0
-	855.900024	0
-	861.300049	0
-	866.700012	0
-	872.100037	0
-	877.500000	0
-	882.900024	0
-	888.300049	0
-	893.700012	0
-	899.100037	0
-	904.500000	0
-	909.900024	0
-	915.300049	0
-	920.700012	0
-	926.100037	0
-	931.500000	0
-	936.900024	0
-	942.300049	0
-	947.700012	0
-	953.100037	0
-	958.500000	0
-	963.900024	0
-	969.300049	0
-	974.700012	0
-	980.100037	0
-	985.500000	0
-	990.900024	0
-	996.300049	0
-	1001.700012	0
-	1007.100037	0
-	1012.500000	0
-	1017.900024	0
-	1023.300049	0
-	1028.700073	0
-	1034.100098	0
-	1039.500000	0
-	1044.900024	0
-	1050.300049	0
-	1055.700073	0
-	1061.100098	0
-	1066.500000	0
-	1071.900024	0
-	1077.300049	0
diff --git a/Runtime/DEMO.exe b/Runtime/DEMO.exe
deleted file mode 100644
index f3e5b00..0000000
Binary files a/Runtime/DEMO.exe and /dev/null differ
diff --git a/Runtime/SCENES/CITY.FLD b/Runtime/SCENES/CITY.FLD
index 134ae61..88ee771 100644
Binary files a/Runtime/SCENES/CITY.FLD and b/Runtime/SCENES/CITY.FLD differ
diff --git a/Runtime/fountain-profiling.txt b/Runtime/fountain-profiling.txt
deleted file mode 100644
index 268c0b3..0000000
--- a/Runtime/fountain-profiling.txt
+++ /dev/null
@@ -1,7 +0,0 @@
-ZCLR 0.0ms (0.0%)
-ANIM 0.0ms (0.0%)
-XFRM 0.0ms (0.0%)
-LGHT 0.0ms (0.0%)
-SORT 0.0ms (0.0%)
-RNDR 17.3ms (89.7%)
-FLIP 2.0ms (10.3%)
diff --git a/Runtime/logfile.txt b/Runtime/logfile.txt
deleted file mode 100644
index 21c3ab0..0000000
--- a/Runtime/logfile.txt
+++ /dev/null
@@ -1,12 +0,0 @@
-] Vertigo GL (V0.85) - INITIALIZE
-] Using MMX(TM) for Rasterization.
-] Hacking Random-seed.
-] Globals initialization.
-] Diffusing Materials.
-] FPU >> Low Precision.
-] Basic font.
-] Initializing Video system.
-Attempting to read: texture file Textures//SC13.JPG as 24 bpp tmap
-Attempting to read: texture file Textures//Code.JPG as 24 bpp tmap
-Attempting to read: texture file Textures//Gfx.JPG as 24 bpp tmap
-Attempting to read: texture file Textures//Sfx.JPG as 24 bpp tmap
diff --git a/fmod/FMod.vcxproj b/fmod/FMod.vcxproj
index a9d21cf..93b3948 100644
--- a/fmod/FMod.vcxproj
+++ b/fmod/FMod.vcxproj
@@ -22,7 +22,7 @@
     <SccProjectName />
     <SccLocalPath />
     <ProjectGuid>{AA8348A9-B816-0319-2AFF-38B85BC354C4}</ProjectGuid>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.10586.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
@@ -242,4 +242,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
